examples/interaction/tooltips/tooltip_content.py

```diff
@@ -1,3 +1,12 @@
+''' A visualization of inputs through Tooltip in bokeh.models.
+This example demonstrates defining two distinct tooltip renderers,
+i.e. plaintext and html tooltip, and accepting text input through those renderers.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.io.show, bokeh.models.TextInput, Bokeh.models.Tooltip
+    :refs: :ref:`ug_interaction_tooltips_tooltip_content`
```

Comment:
@KesharwaniArpita This reference does not exist in the current ReST source for the docs, and this PR does not appear to add it, either.

Reply:
@bryevdv .Should i remove it then? or replace it with [:ref:`ug_interaction_widgets_examples_textareainput`](https://github.com/bokeh/bokeh/blob/branch-3.2/docs/bokeh/source/docs/user_guide/interaction/tooltips.rst#id41)?

Reply:
The example is really focused on tooltips, so I'd suggest `ug_interaction_tooltips` The section heading is a little more general than is ideal but at least it lands in the right general area.

Reply:
@bryevdv , I have done that and also made the suggested changes, is this pr good to go?